frontend/src/components/adminTask.module.css

```diff
@@ -0,0 +1,24 @@
+.backDrop {
+  background-color: rgba(0, 0, 0, 0.5);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  backdrop-filter: blur(2px);
+  z-index: 90;
+}
+.dialog {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  max-width: 100%;
```

Comment:
```suggestion
  width: 92vw;
  max-width: 800px;
```

Reply:
Oh okay that doesnt work with html dialogs... hmm But it works sort of okay 
![Uploading image.png…]()